### ISSUE TITLE
Text me this call number

### ISSUE THREAD

**Issue by gibsonjc** [CARLI/vufind#65]
The "Text me this call number" function available in VuFind 0.6 is not present.

I think a user should ... have the option to have the call numbers texted to them so they can find the items in the stacks.

**Comment by bgant** [comment]
TRN - We have students who are fans of the ‘Text me this call number’ feature in the current catalog. Can something like it be available in the new version?

**Comment by dmcmpbll** [comment]
Unless I misunderstand the question, I believe that this functionality is available already. I just used the "Text This" link highlighted at the top of the first screenshot, and the text did come through to my cellphone (second screenshot).

![textthiscallnumber](https://cloud.githubusercontent.com/assets/20848151/17573388/af68dbdc-5f1f-11e6-89ec-8a950b6e3f54.PNG)

![textthiscallnumber2](https://cloud.githubusercontent.com/assets/20848151/17573443/055570aa-5f20-11e6-9520-dbf0bc3979cc.PNG)

**Comment by dmcmpbll** [comment]
See issue #154 about the text this feature not sending a full URL to get back to the item record.

**Comment by gibsonjc** [comment]
I agree that VuFind 3's default "Text this" function is present and works mostly right, but I think the lack of "Text me THIS call number" function will be most obvious in the I-Share catalog when there are multiple holdings for multiple institutions.  Which institution's call number shows up in that case if you use the default "Text this" function?

**Comment by gibsonjc** [comment]
MIL - Would still like language to be “text call number” and think this feature would be more noticeable next to the actual call number.
